test/NuGet.Licenses.Tests/LicenseFileServiceFacts.cs

```diff
@@ -0,0 +1,132 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Moq;
+using System;
+using System.IO;
+using Xunit;
+
+namespace NuGet.Licenses.Tests
+{
+    public class LicenseFileServiceFacts
+    {
+        private static string TestLicenseIdentifier = "MIT";
+        private static string TestLicenseContent = "MIT license content";
+        private static string TestLicensesFolderPath = "TestFolder\\TestPath\\";
+
+        private static ILicenseFileService CreateService(
+            Mock<ILicensesFolderPathService> licensesFolderPathService = null,
+            Mock<IFileService> fileService = null)
+        {
+            if (licensesFolderPathService == null)
+            {
+                licensesFolderPathService = new Mock<ILicensesFolderPathService>();
+                licensesFolderPathService
+                    .Setup(x => x.GetLicensesFolderPath())
+                    .Returns(TestLicensesFolderPath);
+            }
+
+            if (fileService == null)
+            {
+                fileService = new Mock<IFileService>();
+                fileService
+                    .Setup(x => x.GetFileFullPath(It.IsAny<string>()))
+                    .Returns(Path.Combine(TestLicensesFolderPath, TestLicenseIdentifier, ".txt"));
+                fileService
+                    .Setup(x => x.DoesFileExist(It.IsAny<string>()))
+                    .Returns(true);
+                fileService
+                    .Setup(x => x.ReadFileContent(It.IsAny<string>()))
+                    .Returns(TestLicenseContent);
+            }
+
+            return new LicenseFileService(licensesFolderPathService.Object, fileService.Object);
+        }
+
+        [Fact]
+        public void CheckLicenseFileExists()
+        {
+            // Arrange
+            var newService = CreateService();
+
+            // Act
+            var doesFileExist = newService.DoesLicenseFileExist(TestLicenseIdentifier);
+
+            // Assert
+            Assert.True(doesFileExist);
```

Comment:
Need to verify that `LicenseFileService` makes proper calls to underlying services (`ILicensesFolderPathService`, `IFileService`) in each test. Please add appropriate `Verify` calls

Reply:
Add it now.